woocommerce/woocommerce-ios
# [Settings] Show WooCommerce version and available updates in settings

Closes: #7778


### Description


Adds a `WooCommerce Version` row in Settings, showing the user whether their site has an up-to-date WooCommerce plugin or not, and if not, linking them to the wp-admin screen to update it.

### Testing instructions


On a store with an outdated copy of WooCommerce

1. Navigate to Hub Menu > ⚙️
2. Observe that the WooCommerce Version row has the correct version number.
3. Observe that `Update available` is shown, along with the version number
4. Tap the row and log in to your wp-admin
5. Update WooCommerce
6. Tap done
7. Observe that the row shows `Up to date` after the update is complete

N.B. you can simulate the above by putting a Charles/Proxyman breakpoint on the `/wc/v3/system_status` response, and editing the `woocommerce/woocommerce.php` plugin's `version` attribute to be lower than the `versionLatest`.

### Screenshots


https://user-images.githubusercontent.com/2472348/192614971-bdfa8443-c232-4353-b671-80f5106b86c2.mp4



---
